project/Dependencies.scala

```diff
@@ -19,7 +19,7 @@ object Dependencies {
   // check agrona version when updating this
   val aeronVersion = "1.22.1"
   // needs to be inline with the aeron version
-  val agronaVersion = "1.0.8"
+  val agronaVersion = "1.0.10"
```

Comment:
looks like we should only update this when we update aeron - I'll PR a scala-steward exclusion